dockers/docker-base-arm64/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2016 Microsoft, Inc.
```

Comment:
Why there are 4 license files in this PR? Which part of this PR is licensed?

Reply:
Since I copied it from docker-base, I retained it.
Suggest me so that I can remove those files.

Reply:
Removed.